### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by DemiMarie-parity**
The ones that cannot be upgraded are parity-wasm and jsonrpc.

### CODE REVIEW

File: core/rpc/src/system/helpers.rs [hunk 1]
```diff
@@ -51,7 +51,7 @@ pub struct Health {
 }
 
 /// Network Peer information
-#[derive(Debug, PartialEq, Serialize)]
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
```
Comment (bkchr) [inline core/rpc/src/system/helpers.rs]: Why do they now need deserialize?

File: core/rpc/src/system/helpers.rs [hunk 2]
```diff
@@ -37,7 +37,7 @@ pub struct SystemInfo {
 }
 
 /// Health struct returned by the RPC
-#[derive(Debug, PartialEq, Serialize)]
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
```
Comment (bkchr) [inline core/rpc/src/system/helpers.rs]: Same?

File: core/sr-version/src/lib.rs [hunk 3]
```diff
@@ -63,7 +63,7 @@ macro_rules! create_apis_vec {
 /// In particular: bug fixes should result in an increment of `spec_version` and possibly `authoring_version`,
 /// absolutely not `impl_version` since they change the semantics of the runtime.
 #[derive(Clone, PartialEq, Eq, Encode)]
-#[cfg_attr(feature = "std", derive(Debug, Serialize, Decode))]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize, Decode))]
```
Comment (bkchr) [inline core/sr-version/src/lib.rs]: Why deserialize?

**Comment by tomaka** [comment]
Updating the patch versions in Cargo.toml is normally totally unnecessary, unless someone broke semver somewhere.

File: core/rpc/src/system/helpers.rs (hunk 1 above)
Comment (DemiMarie-parity) [reply core/rpc/src/system/helpers.rs]: Otherwise, the code did not compile, IIRC.

File: core/sr-version/src/lib.rs (hunk 3 above)
Comment (DemiMarie-parity) [reply core/sr-version/src/lib.rs]: I think I got compile errors otherwise.

File: core/rpc/src/system/helpers.rs (hunk 1 above)
Comment (bkchr) [reply core/rpc/src/system/helpers.rs]: Could you please post the error?

File: node/runtime/src/lib.rs [hunk 4]
```diff
@@ -58,8 +58,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 75,
-	impl_version: 75,
+	spec_version: 76,
```
Comment (bkchr) [inline node/runtime/src/lib.rs]: Is not required.

**Comment by gavofyork** [comment]
@DemiMarie-parity is this likely to be resurrected?

**Pull request closed without merging**